main.tf

```diff
@@ -55,6 +62,19 @@ resource "azurerm_subnet" "graphdb-vmss" {
 
 # ------------------------------------------------------------
 
+# Creates a public IP address with assigned FQDN from the regional Azure DNS
+module "address" {
+  source = "./modules/address"
+
+  resource_name_prefix = var.resource_name_prefix
+  resource_group_name  = azurerm_resource_group.graphdb.name
+  zones                = var.zones
+
+  tags = local.tags
+
+  depends_on = [azurerm_resource_group.graphdb]
```

Comment:
Otherwise you'd get

```
│ Error: Error: Resource Group "laina-graphdb" was not found
│ 
│   with module.address.data.azurerm_resource_group.graphdb,
│   on modules/address/main.tf line 1, in data "azurerm_resource_group" "graphdb":
│    1: data "azurerm_resource_group" "graphdb" {
```

which I guess is what you mean with https://github.com/Ontotext-AD/terraform-azure-graphdb/pull/7#discussion_r1397452939

Reply:
No point in refactoring them now. Will do it in another PR

Reply:
Done in [!13](https://github.com/Ontotext-AD/terraform-azure-graphdb/pull/13)